### PR TITLE
fix: highlight security report page

### PR DESCRIPTION
**Pull request by ffe9f8**

### CODE REVIEW

File: src/pages/resources/security.mdx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Bug bounty
```
Comment (StephenFluin) [inline src/pages/resources/security.mdx:1]: Do we want to leave the term bug bounty anywhere? How do other teams approach this and label it? I think some folks will be looking for bounty.

File: src/layouts/MainLayout.astro [hunk 2]
```diff
@@ -150,7 +150,7 @@ const shownDescription = description || 'The Axelar Documentation tells you how
           </div>
           <div>
             <strong>Others</strong><br/>
-            <div><a href="/bug-bounty">Bug Bounty</a></div>
+            <div><a href="/bug-bounty">Report a Vulnerability</a></div>
```
Comment (StephenFluin) [inline src/layouts/MainLayout.astro:153]: Link is wrong with the other changes

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: vercel.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
     },
     {
       "source": "/resources/bug-bounty",
-      "destination": "/resources/community/bug-bounty",
```
Comment (milapsheth) [inline vercel.json:26]: should a redirect be added for this page as well?

File: src/pages/resources/security.mdx (hunk 1 above)
Comment (milapsheth) [reply src/pages/resources/security.mdx:1]: Yeah, reporters search for this. We should preserve that (maybe add it in brackets, or add another reference using it that links to this page). Can you also verify if this page is easily reachable via google search?

Comment (ffe9f8) [reply src/pages/resources/security.mdx:1]: I did a search for the following terms on both my work and personal laptops:

- "axelar bug bounty"
- "axelar report security bug"
- "axelar report security risk"
- "axelar report security"
- "axelar report security vulnerability"

This page didn't pop up on any of those searches. Most of them led directly to our [Immunefi page](https://immunefi.com/bounty/axelarnetwork/), or to the ["Security on Axelar Network"](https://axelar.network/blog/security-at-axelar-core) blog post.

I'm ok with putting "Bug Bounty" in brackets. Will do this now.

File: src/layouts/MainLayout.astro (hunk 2 above)
Comment (ffe9f8) [reply src/layouts/MainLayout.astro:153]: Good catch.

File: vercel.json (hunk 3 above)
Comment (ffe9f8) [reply vercel.json:26]: Added one just in case.

**Comment by StephenFluin** [comment]
You should do 15 minutes of research and look at other teams, but I see bug
bounties more often than "report a vulnerability".

**Comment by ffe9f8** [comment]
Spoke to @benjamin852 and decided to keep the "Bug bounty" name. Clean changes in [PR 807](https://github.com/axelarnetwork/axelar-docs/pull/807).

**Pull request closed without merging**